code/modules/client/preferences_toggles.dm

```diff
@@ -338,6 +338,14 @@
 	else
 		to_chat(src, SPAN_BOLDNOTICE("Dual-wielding now fires both guns simultaneously."))
 	prefs.save_preferences()
+	
+/client/proc/toggle_middle_mouse_swap_hands() //Toggle whether middle click swaps your hands
+	prefs.toggle_prefs ^= TOGGLE_MIDDLE_MOUSE_SWAP_HANDS
+	if(prefs.toggle_prefs & TOGGLE_MIDDLE_MOUSE_SWAP_HANDS)
+		to_chat(src, SPAN_BOLDNOTICE("Middle Click will now swap your hands."))
+	else
+		to_chat(src, SPAN_BOLDNOTICE("Middle Click will no longer swap your hands."))
```

Comment:
these two to_chats can be condensed into one line

Reply:
done